src/VSIX/NpgsqlConnectionEquivalencyComparer.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.VisualStudio.Data.Services.SupportEntities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Npgsql.VSIX
+{
+    class NpgsqlConnectionEquivalencyComparer : IVsDataConnectionEquivalencyComparer
```

Comment:
I think it is good idea to extend `DataConnectionEquivalencyComparer` and override `AreEquivalent`.

`IVsDataConnectionProperties` also implements a dictionary `IDictionary<string, object>`, so we can compare limited values. I'll try it.